### DISCUSSION TITLE
HLS4ML IN QUARTUS SUPPORT FOR CONV2D

### DISCUSSION THREAD

**Discussion by MODISENEISO** [fastmachinelearning/hls4ml#875] [General]
Hi. I am converting a model that has CONV2D layers with QUartus as backend. I get able to get the C++ firmware. However I get this error when attempting synthesis to RTL;

firmware/nnet_utils/nnet_conv2d_stream.h:135:5: error: no matching function for call to 'shift_line_buffer_2d'
    nnet::shift_line_buffer_2d<data_T, CONFIG_T>(in_elem, line_buffer, shift_buffer);

**Comment by vloncar** [comment]
Continued in #876.
